lib/pages/new_group/new_group.dart

```diff
@@ -59,33 +51,43 @@ class NewGroupController extends State<NewGroup>
   @override
   void initState() {
     super.initState();
-    initSearchContacts();
     listenGroupNameChanged();
     _registerListenerForSelectedImagesChanged();
-    scrollController.addLoadMoreListener(loadMoreContacts);
   }
 
   @override
   void dispose() {
     super.dispose();
-    disposeSearchContacts();
-    scrollController.dispose();
     imagePickerController.dispose();
-    selectedContactsMapNotifier.dispose();
     haveGroupNameNotifier.dispose();
     avatarNotifier.dispose();
     uploadContentInteractorStreamSubscription?.cancel();
     createNewGroupChatInteractorStreamSubscription?.cancel();
   }
 
+  @override
+  String getTitle(BuildContext context) {
+    return L10n.of(context)!.newGroupChat;
+  }
+
+  @override
+  String getHintText(BuildContext context) {
+    return L10n.of(context)!.whoWouldYouLikeToAdd;
+  }
+
+  @override
+  List<String> get disabledContacts => [];
```

Comment:
What is disabledContacts, and you shoudn't override like this if it don't have meaning in this class, maybe don't need to abstract this method,